src/results.js

```diff
@@ -6,11 +6,34 @@ function timeMs() {
   return new Date().getTime()
 }
 
-function Preview({url}) {
-  let embedUrl = `${url}?embed=on`
-  return (
-    <iframe className='results-item-preview' src={embedUrl} />
-  )
+class Preview extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {loaded: false}
```

Comment:
You can do it simpler: don't set `this.state` in constructor, and below, accept the fact that `this.state` might be undefined:
```javascript
(this.state || {}).loaded // at first this evaluates to undefined, then to true
```